packages/read-emails/frontend/react/src/components/Layout.jsx

```diff
@@ -2,22 +2,33 @@ import React from 'react';
 import IconSync from './icons/icon-sync.svg';
 import IconLogout from './icons/icon-logout.svg';
 import NylasLogo from './icons/nylas-logo-horizontal.svg';
+import PropTypes from 'prop-types';
+
+const Layout = ({ children, showMenu = false, disconnectUser, refresh }) => {
+  const handleRefresh = (e) => {
+    e.preventDefault();
+    refresh();
+  };
+
+  const handleDisconnect = (e) => {
+    e.preventDefault();
+    disconnectUser();
```

Comment:
check what product wants to do, but I think we need to put this in a `setTimeout` and add an animation...it's too fast!  Can add as part of the clean up epic.

Reply:
👍